src/base32.zig

```diff
@@ -95,12 +95,14 @@ const Encoding = struct {
                 dst[5] = self.buf[b[5] & 31];
                 dst[6] = self.buf[b[6] & 31];
                 dst[7] = self.buf[b[7] & 31];
+                n += 8;
             } else {
                 var i: usize = 0;
                 while (i < size) : (i += 1) {
                     const x = b[i] & 31;
                     dst[i] = self.buf[b[i] & 31];
                 }
+                n += i;
```

Comment:
I'm not sure I got this right. This branch doesn't seem to be covered by any of the tests.
<img width="612" alt="Screen Shot 2021-11-28 at 11 38 52 PM" src="https://user-images.githubusercontent.com/145012/143826869-f48b9829-bae2-45c1-ade9-25b989e80b7d.png">